functions/functions/voice/update-in-progress-call.js

```diff
@@ -0,0 +1,13 @@
+const { updateInProgressCall } = require(Runtime.getFunctions()["twilio/voice"].path);
+
+exports.handler = (context, event, callback) => {
+    const client = context.getTwilioClient();
+    const { callSid, twiml } = event;
+
+    updateInProgressCall(client, callSid, twiml)
+    .then(data => callback(null, data))
+    .catch(err => {
+        console.log(`Error updating in progress call ${err}`);
```

Comment:
same as above console.log in functions comment

Reply:
Yolo